website/docs/r/pi_shared_processor_pool.html.markdown

```diff
@@ -22,7 +24,8 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
 }
 ```
 
-**Note**
+## Notes
```

Comment:
Please double check this when you get a chance, but I think ### was equivalent to **. I recall there being some size differences when I tried it yesterday.

```suggestion
### Notes
```